### ISSUE TITLE
don't make plus-signs selectable

### ISSUE THREAD

**Issue by SimonSiefke** [webpack/webpack.js.org#3578]
### Expected behavior: 
I can copy and paste the code, the plus-signs are not copied with the code.

### Current behavior:
When I copy and paste the code, I manually need to remove each `+` because the the plus-signs are copied along with the code.

![webpack-select](https://user-images.githubusercontent.com/23744935/75109201-8300d200-5621-11ea-9ec4-cc61dd659b9c.png)

**Comment by anikethsaha** [comment]
Those `+` and `-` signs are required in order to make it better to understand how to change the code for migration or how to replace any deprecated APIs with the new one. 

these are not usually long snippets so manually deleting should not be a much of work. 👍

**Comment by SimonSiefke** [comment]
I was thinking that maybe 
```html
<p><span style="user-select:none">+</span>document.body.removeChild(element);</p>
<p><span style="user-select:none">+</span>element = component();</p>
<p><span style="user-select:none">+</span>document.body.appendChild(element);</p>
```

would be a good solution. 

I agree that it is not much work to manually remove the plus-signs but I think it is unnecessary and that the guides would be improved when the code can just be copied and pasted.

**Comment by chenxsan** [comment]
@SimonSiefke  It could be possible if we bring Prismjs plugin https://prismjs.com/plugins/diff-highlight/ support into [Refactor](https://github.com/wooorm/refractor)  which this doc site relies one.

**Comment by DanielRuf** [comment]
Or render `:before` pseudo elements.

I think this issue can be closed as https://github.com/PrismJS/prism/pull/2281 is merged?

**Comment by chenxsan** [comment]
@DanielRuf That commit is not released in a new version PrismJS yet. Maybe we can keep this issue open if we do want to bring this feature into doc site.

**Comment by chenxsan** [comment]
- [x] Wait for new PrismJS to be released ([1.21.0](https://github.com/PrismJS/prism/releases/tag/v1.21.0) is out)
- [ ] Wait for [refractor](https://github.com/wooorm/refractor) to update to PrismJS 1.21.0
- [ ] Wait for [remark-refractor](https://github.com/montogeek/remark-refractor) to update to then-latest refractor

**Comment by chenxsan** [comment]
I've created a pull request here https://github.com/webpack/webpack.js.org/pull/3912 so anyone can check how it looks like now https://webpack-js-org-git-fork-chenxsan-feature-user-select-none-diff.webpack-docs.vercel.app/guides/asset-management/#loading-images.
